src/vaishnaviVJun20_arrayExamples/Prime.java

```diff
@@ -0,0 +1,66 @@
+package vaishnaviVJun20_arrayExamples;
+
+public class Prime {
+	
+	int count;
+	int sum;
+	void isPrime(int input) {
+		boolean flag = true;	
+		 for(int index=2;index<=input/2;index++) {	
+		   if(input%index==0) {
+			 flag= false;
+			 System.out.println(input+" is not a prime number");
+			 break;
+		   }
+		}
+		if(flag == true)
+			System.out.println(input +" is a Prime number");
+	}
+	
+	void findPrimeNumbers() {
```

Comment:
This method should be parameterized. If tomorrow I want to use this method for range 100 to 1000, this won't work.

Reply:
Done suggested changes.